### ISSUE TITLE
Bug: commits with just the title crash

### ISSUE THREAD

**Issue by Kixunil** [crywolf/mkchlog#12]
```
Setup CI

changelog:
    section: dev
    inherit: title
    title-is-enough: true
```

This should just copy the title and not worry about description. It should still fail if there's `inherit: all` of course.

**Comment by crywolf** [comment]
Fixed by be416a0e07decf478cf92c1f14888a90feba617d

**Comment by crywolf** [comment]
Fixed by be416a0e07decf478cf92c1f14888a90feba617d
